plugins/MultiSites/Dashboard.php

```diff
@@ -257,41 +263,39 @@ private function makeCloneOfDataTableSites(DataTable $sites)
      *    site7
      * )
      *
-     * @param $sites
-     * @return array
+     * in a sorted order
+     *
+     * @param DataTable $table
+     * @param array $request
      */
-    private function makeSitesFlat($sites)
+    private function makeSitesFlatAndApplyGenericFilters(DataTable $table, $request)
     {
-        $flatSites = array();
+        // we handle limit here as we have to apply sort filter, then make sites flat, then apply limit filter.
+        $filterOffset = $request['filter_offset'];
+        $filterLimit  = $request['filter_limit'];
+        unset($request['filter_offset']);
+        unset($request['filter_limit']);
 
-        foreach ($sites as $site) {
-            if (!empty($site['subtable'])) {
```

Comment:
All this code is no longer needed since we remove all subtables in `nestedLimit` filter